app/controllers/concerns/saml_idp_logout_concern.rb

```diff
@@ -21,13 +21,19 @@ def handle_valid_sp_logout_request
     sign_out if user_signed_in?
   end
 
-  def handle_valid_sp_remote_logout_request(user_id)
+  def handle_valid_sp_remote_logout_request(user_id:, issuer:)
     # Remotely delete the user's current session
     session_id = ServiceProviderIdentity.
-      find_by(user_id: user_id, service_provider: saml_request.issuer).
-      rails_session_id
+      where(user_id: user_id, service_provider: saml_request.issuer).pick(:rails_session_id)
```

Comment:
[question] is `saml_request.issuer` different from the `issuer` passed in?

Reply:
whoops, it shouldn't be, but I intended to replace `saml_request.issuer` with the argument passed to the function

Reply:
Fixed in 5f24ce655